PlutoHelperAgent/ProjectLockerAndKeychainFunctions.m

```diff
@@ -108,7 +108,7 @@ + (NSDictionary *) load_data_from_keychain {
 + (NSURLSessionTask *) communicate_with_server:(NSString*)url :(NSString*)method :(NSString*)type :(NSDictionary*)body :(BOOL)send_cookie :(BOOL)test_connection
                          completionHandler:(void (^) (NSURLResponse*, NSDictionary *))completionHandlerBlock {
   
-    NSString *URLToUse = [NSString stringWithFormat: url, [[NSUserDefaults standardUserDefaults] stringForKey:@"project_locker_url"]];
+    NSString *URLToUse = [NSString stringWithFormat: @"%@%@", [[NSUserDefaults standardUserDefaults] stringForKey:@"project_locker_url"], url];
```

Comment:
I think that there is a builtin method in NSString to combine filesystem and URL paths - https://developer.apple.com/documentation/foundation/nsstring/1417198-pathwithcomponents?language=objc which might be better here, avoiding any problems with missing or doubling / separators

Reply:
That is not what is being done here so it is not relevant.